### PR TITLE
Recipe for Forked Spear

### PR DESCRIPTION
**Pull request by Dozed12**
- Now uses 3 spikes instead of 3 forks, more info at #11314

### CODE REVIEW

**Comment by BevapDin** [comment]
For future PRs: You should put the item definition and the recipes into one PR. An item definitions itself does not make the item appear and therefor the player will not be able to get it.

You can either add the item to an item list, so it spawns during map generation or as part of monster death drops, or add a recipes that produces the item (as you did). But without any of those sources, the item will be effectively unavailable.


PS: Welcome to the open source community, Dozed12. (-:

**Comment by Dozed12** [comment]
Indeed I know people do that, but I'm still not familiar how github works and I didn't knew how to put both requests in one. 

Thanks for the welcoming message.

**Comment by KA101** [comment]
Simply make all the forkspear-y changes in the same Git branch (here, patch-4) and they'll all go as one.  You can change as many files as you like in a given branch.

File: data/json/recipes/recipe_weapon.json [hunk 1]
```diff
@@ -279,6 +279,44 @@
   ]
 },{
   "type" : "recipe",
+  "result": "spear_forked",
+  "category": "CC_WEAPON",
+  "subcategory": "CSC_WEAPON_PIERCING",
+  "skill_used": "fabrication",
+  "skills_required": [["survival", 2 ],["fabrication", 1]],
```
Comment (KA101) [inline data/json/recipes/recipe_weapon.json]: OK.  Fab as the skill_used implicitly requires it at the difficulty.  No need to have it as a skills_required.

**Comment by KA101** [comment]
This looks closer to the spear_knife than the javelin, so I'm going to change it a little closer to the knife one in the merge.

**Comment by Dozed12** [comment]
Again, thanks, I will be more careful with organizing the code.

**Pull request merged**